beta.Dockerfile

```diff
@@ -1,18 +1,9 @@
 # From https://github.com/docker-library/docs/blob/master/eclipse-temurin/README.md#using-a-different-base-image
 FROM node:20.15.0-bullseye-slim AS build-node
 ENV LANG=en_US.UTF-8
-ENV JAVA_HOME=/opt/java/openjdk
-COPY --from=eclipse-temurin:11 $JAVA_HOME $JAVA_HOME
-ENV PATH="${JAVA_HOME}/bin:${PATH}"
 
 RUN apt-get update && apt-get install -y git graphviz wget fonts-dejavu fontconfig make
```

Comment:
You can make test with minimal package?
IIRC `graphviz` and `wget` was for plantuml, maybe others too

Reply:
thanks for the catch

Reply:
thanks for the catch